frontend/app/components/panel/tile/WithTeam.vue

```diff
@@ -3,7 +3,29 @@ const props = defineProps<{
   teamName: string
 }>()
 
-const value = ref(0)
+const { data } = useAuth('/teams/membership/completed_tasks', {
```

Comment:
you are probably just missing an await statement that may result in undefined behaviour